apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -889,15 +1023,15 @@ mod router_plugin {
 }
 
 #[derive(Default)]
-pub(crate) struct RhaiExecutionResponse {
+pub(crate) struct RhaiRouterResponse {
     pub(crate) context: Context,
-    pub(crate) response: http_ext::Response<Response>,
+    pub(crate) response: http::Response<Body>,
 }
 
 #[derive(Default)]
-pub(crate) struct RhaiExecutionDeferredResponse {
+pub(crate) struct RhaiRouterDeferredResponse {
     pub(crate) context: Context,
-    pub(crate) response: Response,
+    pub(crate) response: Body,
```

Comment:
It's just about preserving symmetry. I know the normal router response is a Response<Body>, so I maintain that a deferred response is a Body. It feels a bit like using Bytes would be relying on an implementation side effect, which doesn't deliver a lot of benefit, since the main overhead of treating Body as Bytes is the `clone()`.